server/conn.go

```diff
@@ -1117,6 +1117,7 @@ func (cc *clientConn) Run(ctx context.Context) {
 					zap.String("status", cc.SessionStatusToString()),
 					zap.Stringer("sql", getLastStmtInConn{cc}),
 					zap.String("txn_mode", txnMode),
+					zap.Uint64("timestamp", cc.ctx.TxnInfo().StartTS),
```

Comment:
The `ctx` could be nil if the session opeing fails, also the returned `TxnInfo` could be nil if the `txn` is not activiated.

Reply:
> The `ctx` could be nil if the session opeing fails, also the returned `TxnInfo` could be nil if the `txn` is not activiated.

would  `ctx` be nil ？ if yes, then the next line also has a bug.